Update "Check ClusterIngress/Istio routing" in debugging-application-… (#2049)
knative/docs: docs/serving/debugging-application-issues.md

The `conditions` in `status` provide the reason if there is any failure. For
details, see Knative
[Error Conditions and Reporting](https://github.com/knative/docs/blob/master/docs/serving/spec/knative-api-specification-1.0.md#error-signalling)(currently

> some of them are not implemented yet).

Reviewer: "(currently some of them are not implemented yet)"

What does this mean (which ones)? The spec does not elaborate so it's unclear what is being referenced. Can this statement just be removed?

Author: I think the `severity` in error conditions is not been implemented.

Author: I read the `error-signalling` section and the relevant code. I am not find which one is not implemented. May be this could be removed.

### Check Ingress/Istio routing

> Run the following command to list all the ingress, with their labels:

Reviewer: ```suggestion
To list all Ingress resources and their corresponding Routes, run the following command:
```

```shell
kubectl get ingresses.networking.internal.knative.dev -o=custom-columns='NAME:.metadata.name,LABELS:.metadata.labels'
NAME            LABELS
helloworld-go   map[serving.knative.dev/route:helloworld-go serving.knative.dev/routeNamespace:default serving.knative.dev/service:helloworld-go]
```

The labels `serving.knative.dev/route` and `serving.knative.dev/routeNamespace`

> will tell exactly which Route an Ingress is a child resource of. Find the

Reviewer: ```suggestion
indicate the Route in which the Ingress resource resides.
```

> one corresponding to your Route. If an Ingress does not exist, the route

Reviewer: ```suggestion
Your Route and Ingress should be listed. If your Ingress does not exist, the route
```

controller believes that the Revisions targeted by your Route/Service isn't
ready. Please proceed to later sections to diagnose Revision readiness status.

Otherwise, run the following command to look at the ClusterIngress created for
your Route

```
kubectl get ingresses.networking.internal.knative.dev <INGRESS_NAME> --output yaml
```

particularly, look at the `status:` section. If the Ingress is working
correctly, we should see the condition with `type=Ready` to have `status=True`.
Otherwise, there will be error messages.

> You can replace resource `ingresses.networking.internal.knative.dev` with `ClusterIngress` if you are using Knative release older than 0.9.

Reviewer: We do not need to reference past versions. Our content is published by release and therefore, users of past versions should view the previous/corresponding doc set release.
```suggestion
```

> Now, if Ingress shows status Ready, there must be a corresponding

Reviewer: ```suggestion
Now, if Ingress shows status `Ready`, there must be a corresponding
```
